README.md

```diff
@@ -2265,6 +2272,47 @@ AccordionItem default expand
 </Accordion>
 
 ```
+### ListBlock
+
+Used to entails the presence of two distinct columns, separated by a divider with each column presenting a list of items in ordered or unordered.
+
+![listblock](docs/images/list-block.png)
+
+```
+<ListBlock slots="left, right" repeat="4" list="unorder" iconColor="#2ac3a2" icon="checkmark" variant="fullWidth" />
+
+500 free Document Transactions per month
+
+Volume and multi-product discounts
+
+Access to all 15+ PDF Services including PDF Extract, PDF Accessibility Auto-Tag API, and Document Generation
+
+Access to all 15+ PDF Services including PDF Extract, PDF Accessibility Auto-Tag API, and Document Generation
+
+Easy to sign up and create credentials in minutes
+
+Technical Support included (different tiers available)
+
+No credit card or commitment required
+
+Scalable for high volume needs.
+```
+
+Use `slots` to identify the markdown content:
+
+- `left`(required)
+
+- `right`(required)
+
+Use `repeat` to define how many code sections are part.
```

Comment:
'repeat' seems to be a required.  Please make sure this is added in readme.

Reply:
I added as required